ls01b_node:
  ros__parameters:
    scan_topic: scan       #设置激光数据topic名称
    frame_id: laser_link     #激光坐标
    serial_port: /dev/ttyUSB0  #雷达连接的串口
    baud_rate: 460800        #雷达连接的串口波特率
    angle_resolution: 0.25     #雷达角度分辨率
    rpm: 600                 #　雷达转速
    angle_compensate: true                 #　角度补偿，非同轴
    robot_radius: 0.22 # 机器人半径
    center_x: 0.0   # 雷达坐标系下机器人中心位置
    center_y: 0.1
    truncated_mode: 0 # 0: 不使用过滤 ;1 :按角度过滤; 2 按半径截取； 3： 按角度加半径截取
    special_range_: 0  #无效值设置
    disable_min: 0,30,60,90
    disable_max: 10,40,70,180
